.github/workflows/go.yml

```diff
@@ -51,7 +51,7 @@ jobs:
       - name: Build executable
         run: make
 
-      - name: Run attestation command integration Tests
+      - name: Run attestation command set integration tests
+        shell: bash
```

Comment:
We can revisit running tests on Windows with powershell as well, but to resolve this bug, we can just start with bash for now. The colon special character issue is not specific to powershell or bash on Windows.